src/guis/yarpmotorgui/main.cpp

```diff
@@ -125,15 +125,16 @@ int main(int argc, char *argv[])
             adr = Network::queryName(descLocalName);
         }
 
-        if (yarp::os::Network::exists("/robotDescription/rpc"))
+        std::string remote_port = "/yarpRobotInterface/devices/rpc";
```

Comment:
Could it make sense to have  this name be configured? We have scenarios in which we have multiple robots in the same YARP network, and so probably we would open the `robotDescription_nws_yarp` with a robot-specific prefix. 

fyi @S-Dafarra @giotherobot @carloscp3009

Reply:
Absolutely yes

Reply:
added parameter `remoteRobotDescriptionPort`